### PR TITLE
EOS-19795: index lookup hints

### PR DESCRIPTION
**Pull request by evgeniy-brazhnikov**
Signed-off-by: evgeniy-brazhnikov <evgeniy.brazhnikov@seagate.com>

### CODE REVIEW

File: st/clitests/backgrounddelete_spec.py [hunk 1]
```diff
@@ -140,8 +140,8 @@ def perform_head_object(oid_dict):
 load_and_update_config(account_response_elements['AccessKeyId'], account_response_elements['SecretKey'])
 
 #Clear probable delete list index
-s3kvs.clean_all_data()
-s3kvs.create_s3root_index()
+#s3kvs.clean_all_data()
+#s3kvs.create_s3root_index()
```
Comment (knrajnambiar76) [inline st/clitests/backgrounddelete_spec.py:144]: why these commented ?

**Review by knrajnambiar76 — COMMENTED** [review]
In s3backgrounddelete's config file (config.yaml) within indexid section  some of the global index's OID is hardcoded there, that makes s3background delete aware of the index OIDs.
This will change right
orfor global index, will old mechanism be allowed

File: st/clitests/backgrounddelete_spec.py (hunk 1 above)
Comment (evgeniy-brazhnikov) [reply st/clitests/backgrounddelete_spec.py:144]: These lines recreate global indices after s3server creates them and saves their pool version id & layout type.

**Comment by gowthamchinna** [comment]
test this please

File: st/clitests/backgrounddelete_spec.py [hunk 2]
```diff
@@ -805,8 +805,8 @@ def get_upload_id(response):
    .execute_test().command_is_successful()
 
 #Clear probable delete list index
-s3kvs.clean_all_data()
-s3kvs.create_s3root_index()
+#s3kvs.clean_all_data()
+#s3kvs.create_s3root_index()
```
[the remark this answers is not in the document]
Comment (DPG17) [reply st/clitests/backgrounddelete_spec.py:809]: Why the calls to clean all data and re-creating data are not commented in other ST spec files?

File: st/clitests/s3cmd_spec.py [hunk 3]
```diff
@@ -61,6 +61,8 @@
 Config.config_file = "pathstyle.s3cfg"
 
 # ************ Test post removal of data (removal/creation of root index) ************
+# Recreating of global indices causes s3server can loose access to them.
+'''
 s3kvs.clean_all_data()
```
Comment (DPG17) [inline st/clitests/s3cmd_spec.py:66]: As I mentioned earlier, we have s3kvs.clean_all_data() is many ST spec file. This is another place that we have this call. Why this is not commented (like in BD ST spec file)?

**Review by DPG17 — APPROVED** [review]

**Review by knrajnambiar76 — APPROVED** [review]
Please put some details about the change in commit message

**Comment by nileshgovande** [comment]
Ceph and splunk has passed - 
http://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/S3Server/job/S3-Compatibility-Test-Splunk/236/console
http://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/S3Server/job/S3-Compatibility-Test-Ceph/244/console

**Pull request merged**